src/ImageEnhanceViaFusion.cpp

```diff
@@ -0,0 +1,118 @@
+#include "ImageEnhanceViaFusion.hpp"
+
+#include <vector>
+#include <iostream>
+
+
+#include <opencv2/imgproc/imgproc.hpp>
+
+namespace underwater_image_tools {
+
+
+// percent should have values between 0.0001 and 0.05
+cv::Mat colorBalance( cv::Mat image, const cv::Scalar& percent_channel ){
```

Comment:
Once you are passing, modifying and returning the same `image`, you can pass this `cv::Mat` as reference and avoid copies.